Makefile

```diff
@@ -61,6 +62,22 @@ unlock-rpm:
 	sudo yum versionlock clear
 
 # Add other package types here.
```

Comment:
Please move the "# Add other package types here." comment after all the packages.
Please add a comment here "#\n# DEB builds\n#\n" similar to the other packages.